app/controllers/state_file/questions/md_tax_refund_controller.rb

```diff
@@ -0,0 +1,19 @@
+module StateFile
+  module Questions
+    class MdTaxRefundController < QuestionsController
+      def self.show?(intake)
+        intake.calculated_refund_or_owed_amount.positive?
+      end
+
+      def refund_amount
+        current_intake.calculated_refund_or_owed_amount
+      end
+      helper_method :refund_amount
+
+
+      private
+
+      def card_postscript; end
+    end
```

Comment:
Nonblocking question: what is this for?

Reply:
honestly I just copied it from the existing taxrefundcontroller